libs/orchestrator-ui-components-storybook/package.json

```diff
@@ -0,0 +1,67 @@
+{
+    "name": "storybook-app",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "@elastic/datemath": "^5.0.3",
+        "@elastic/eui": "^81.0.0",
+        "@emotion/css": "^11.11.0",
+        "@emotion/react": "^11.11.0",
+        "@orchestrator-ui/orchestrator-ui-components": "^0.0.1-alpha.9",
```

Comment:
Is there a way to use the actual current version of the lib? The currect setup requires a publish to NPM before implementing the Storybook parts.